blog/20240516-simplex-redefining-privacy-hard-choices.md

```diff
@@ -0,0 +1,75 @@
+
+To make a profit, satisfying customers is the key. Unlike the many companies that profit from selling customer data, we put user privacy first. Doing this at scale requires investments. If the investors don’t own or control a company, their participation becomes merely about profit for them, and not about how this profit is obtained. With the investors we have, we are completely aligned on this - they are betting on the future where privacy is the norm. They do not dictate on anything related to our model. We build SimpleX chat, protocols and network the way Internet should have been built if we as developers always put the privacy and empowerment of users first.
+
+## Company jurisdiction
+
+<img src="./images/20240516-parliament.jpg" width="40%" class="float-right">
+
+With regards to jurisdictions, nowhere is perfect. For that reason we plan to establish the foundations for protocol governance in [various jurisdictions](https://simplex.chat/blog/20240323-simplex-network-privacy-non-profit-v5-6-quantum-resistant-e2e-encryption-simple-migration.html#the-journey-to-the-decentralized-non-profit-protocol-governance). But we’d like to explain some misconceptions about the UK, where SimpleX Chat Ltd. is registered.
+
+The first common myth is about the the Online Safety Act (OSA) - some people believe that it applies to the UK companies. It is simply incorrect - the OSA applicability isn't determined by the jurisdiction of a company. The OSA applies based on the nature and characteristics of the business and its services, as well as the number of its users in the UK. In case of SimpleX network, the OSA doesn't apply for both of these reasons.
+
+Another myth is about the UK position on communication encryption. The UK is also not anti-encryption. For example, it didn't attempt to ban the encryption; the legislative intent was to propose technical measures to block CSAM. As the discussion about the bill progressed, the parliament amended the bill, and the government publicly committed to not request the implementation of client-side scanning until a viable technology emerges that wouldn't compromise e2e encryption (which at this point appears to be a wishful thinking). The process's intentions and outcome don't look like an attempt to prohibit encryption. This may have been the desire of lobbyists, like certain anti-privacy non-profits, but it wasn't the aim of the UK government per their own communications on the matter. 
```

Comment:
Another myth is about the UK position on communication encryption. The UK government is not anti-encryption. For example, it didn't attempt to have a blanket ban on strong encryption; the legislative intent was to propose technical measures to block CSAM. This has led to recommendations for backdoors into messaging services, which of course could be easily misused for other purposes and is wrong. There have been suggested amendments to force backdoors into messaging services, but those have not passed. As such, as the discussion about the bill progressed, the parliament amended the bill, and the government publicly committed to not request the implementation of client-side scanning until a viable technology emerges that wouldn't compromise e2e encryption (which at this point appears to be a wishful thinking). The process's intentions and outcome don't look like an attempt to prohibit encryption at large. This may have been the desire of lobbyists, like certain anti-privacy non-profits, but it wasn't the aim of the UK government per their own communications on the matter.